rpm/oprofile.spec

```diff
@@ -34,21 +35,20 @@ Requires:   binutils-devel
 Header files and libraries for developing apps which will use oprofile.
 
 %prep
-%setup -q -n %{name}-%{version}/%{name}
+%autosetup -p1 -n %{name}-%{version}/%{name}
 
 %build
 sh autogen.sh
 
-%configure --disable-static \
+%configure --verbose --verbose --disable-static \
```

Comment:
Quite verbose. Is this intentional?

Reply:
Oops, forgot to remove that before making the PR, removed.